release-notes/opensearch-dashboards-search-relevance.release-notes-2.7.0.0.md

```diff
@@ -0,0 +1,9 @@
+## What's Changed
+* Welcome new maintainers, Sean Li and Louis Chu! by @macohen in https://github.com/opensearch-project/dashboards-search-relevance/pull/172
+* Fix Node.js and Yarn installation in CI by @msfroh in https://github.com/opensearch-project/dashboards-search-relevance/pull/156
+* removing stats.yml until an alternate can be found that can publish P… by @macohen in https://github.com/opensearch-project/dashboards-search-relevance/pull/163
+* Instrument metrics on server-side by @noCharger  in https://github.com/opensearch-project/dashboards-search-relevance/pull/164
+* Refactor metric names by @noCharger in https://github.com/opensearch-project/dashboards-search-relevance/pull/176
```

Comment:
shall we apply labels to the PRs?

Reply:
Example format https://github.com/opensearch-project/dashboards-search-relevance/blob/main/release-notes/opensearch-dashboards-search-relevance.release-notes-2.5.0.0.md

Reply:
Will do. There were no labels on most of the PRs. We should make that standard, but I'll go through it tonight. Thanks!

Reply:
categorized the release notes. it seems that the github action that should generate the notes is not working so it couldn't be automatically generated. https://github.com/opensearch-project/dashboards-search-relevance/actions/runs/4737728702/jobs/8410847077